cms/djangoapps/models/settings/course_grading.py

```diff
@@ -104,7 +111,8 @@ def update_cutoffs_from_json(course_key, cutoffs, user):
         descriptor.grade_cutoffs = cutoffs
 
         modulestore().update_item(descriptor, user.id)
-
+        _emit_grading_event(course_key, user.id)
+        GRADING_POLICY_CHANGED.send(sender=CourseGradingModel, user_id=user.id, course_id=course_key)
```

Comment:
@yro @iloveagent57 here's the spot I think we missed the first time around. Updating grade cutoffs can change learners' letter grades, so we should emit the signal here.